kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/settings/ApplicationCertsTabUi.java

```diff
@@ -197,4 +197,10 @@ private boolean isAppCertValid() {
             return true;
         }
     }
+
+    @Override
```

Comment:
Please change the header to: " * Copyright (c) 2011, 2020 Eurotech and/or its affiliates"